assignments/lambda-classes.js

```diff
@@ -1 +1,265 @@
 // CODE here for your Lambda Classes
+
+// Person class (base class)
+
+class Person {
+
+  constructor(props) {
+
+    this.name = props.name;
+    this.age = props.age;
+    this.gender = props.gender;
+    this.location = props.location;
+
+  }
+
+  speak() {
+
+    console.log(`Hello my name is ${this.name}, I am from ${this.location}`);
+
+  }
+
+}
+
+// Instructor class (child of Person)
+
+class Instructor extends Person {
+
+  constructor(props) {
+
+    super(props);
+
+    this.specialty = props.specialty;
+    this.favLanguage = props.favLanguage;
+    this.catchPhrase = props.catchPhrase;
+
+  }
+
+  demo(subject) {
+
+    console.log(`Today we are learning about ${subject}`);
+
+  }
+
+  grade(student, subject) {
+
+    console.log(`${student.name} receives a perfect score on ${subject}`);
+
+  }
+
+  changeGrade(student) {
+
+    let addOrSub = Math.random();
+
+    if (addOrSub > 0.5) {
```

Comment:
It's seems you could forego the variable here since you only use it once in one place (and it doesn't necessarily need any additional clarity):
```javascript
if (Math.random() > 0.5) { ... }
```